StyleCop.Analyzers/StyleCop.Analyzers/DocumentationRules/SA1604ElementDocumentationMustHaveSummary.cs

```diff
@@ -20,6 +20,7 @@
     /// tag.</para>
     /// </remarks>
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
+    [NoCodeFix("Cannot generate documentation")]
```

Comment:
:exclamation: I believe this one can be generated, at least as an element containing no text. The result would then produce warning SA1606, but at least the user wouldn't have to type the XML portion manually.

```csharp
/// <summary>
///
/// </summary>
```